packages/kotti-ui/source/kotti-accordion/KtAccordion.vue

```diff
@@ -46,8 +46,6 @@ export default {
 	},
 	mounted() {
 		this.isOpen = !this.isClosed
-		// This is the best solution right now
```

Comment:
what is this change related to @HansAuger 
can we discard this

Reply:
I’m gonna discard this so we can move on